docs/submitting-clinical-data.md

```diff
@@ -2,15 +2,28 @@
 id: submitting-clinical-data
 title: Submitting Clinical Data
 ---
+Clinical data consists of all supporting data about your programs donors and their health data.
 
-## Uploading Data
+> Note: Before you can submit clinical data, your data must be assigned ARGO Identifiers through the sample registration process. See [Registering Samples](/s) for instructions on how to register samples.
 
-### How To
 
-### Links
+## How To Submit Clinical Data
+1. Navigate to the **Submission** area in the top menu.
+1. Click on the **Clinical Submission** section in the left menu pane for your program.
+1. Download a **TSV Template** for the data type that you are submitting. Format your file(s) according to the [Data Dictionary](/dictionary) specifications for each clinical data type.
```

Comment:
Download a TSV template for each data type you are submitting and format the file(s) according to the current data dictionary specifications for each clinical data type.

(to make consistent with above)